catalog/app/website/pages/Landing/Landing.js

```diff
@@ -24,23 +24,27 @@ import UseQuilt from './UseQuilt'
 export default function Landing({ location }) {
   const cfg = Config.useConfig()
   const { q: query = '' } = parseSearch(location.search)
+  const showMarketingBlocks = cfg.mode !== 'LOCAL' && cfg.mode !== 'PRODUCT'
   return (
     <Layout>
       <MetaTitle />
       <React.Suspense fallback={null}>
         <LinkedData.CatalogData />
       </React.Suspense>
-      {cfg.mode !== 'LOCAL' && <Dots />}
+      {cfg.mode !== 'LOCAL' && (
+        <Dots style={{ height: cfg.mode === 'PRODUCT' ? '1109px' : undefined }} />
+      )}
       {cfg.mode === 'PRODUCT' && <Buckets query={query} />}
-      {cfg.mode === 'LOCAL' ? <LocalMode /> : <Showcase />}
-      {cfg.mode !== 'LOCAL' && <UseQuilt />}
-      {cfg.mode !== 'LOCAL' && <Logos />}
-      {cfg.mode !== 'LOCAL' && <CaseStudies />}
-      {cfg.mode !== 'LOCAL' && <Testimonials />}
-      {cfg.mode !== 'LOCAL' && <Platform />}
-      {cfg.mode !== 'LOCAL' && <Highlights />}
+      {cfg.mode === 'LOCAL' && <LocalMode />}
+      {showMarketingBlocks && <Showcase />}
+      {showMarketingBlocks && <UseQuilt />}
+      {showMarketingBlocks && <Logos />}
+      {showMarketingBlocks && <CaseStudies />}
+      {showMarketingBlocks && <Testimonials />}
+      {showMarketingBlocks && <Platform />}
+      {showMarketingBlocks && <Highlights />}
```

Comment:
we could wrap these into a fragment and keep only one check before it, e.g.:
```
{showMarketingBlocks && (
  <>
    <Showcase />
    <UseQuilt />
    ...
  </>
)}
```
no big deal tho, up to you